package.json

```diff
@@ -12,14 +12,15 @@
   },
   "dependencies": {
     "@graphprotocol/graph-cli": "0.25.1",
-    "@graphprotocol/graph-ts": "0.24.1"
+    "@graphprotocol/graph-ts": "0.24.1",
+    "ethers": "^5.5.3"
```

Comment:
That looks weird, are you sure?

Reply:
When I ran the prepare script I had a "Module not found error" for ethers. It solved it

Reply:
could be listed as a dev dependency (just for clarifying it's only used in scripts, not in the main code, it won't make a difference) so not required!